modules/openapi-generator/src/main/resources/typescript-nestjs/api.service.mustache

```diff
@@ -106,7 +106,7 @@ export class {{classname}} {
 {{#useSingleRequestParameter}}
         const {
         {{#allParams}}
-            {{paramName}},
+            {{#hasSanitizedName}}'{{{baseName}}}': {{/hasSanitizedName}}{{paramName}},
```

Comment:
this is not an object, but destructuring the request parameters, please revert this change